dragonfly/block/gravel.go

```diff
@@ -0,0 +1,27 @@
+package block
+
+import "github.com/df-mc/dragonfly/dragonfly/item"
+
+// Gravel is a block that is affected by gravity.
+type Gravel struct{}
+
+// BreakInfo ...
+func (g Gravel) BreakInfo() BreakInfo {
+	return BreakInfo{
+		Hardness:    0.6,
+		Harvestable: alwaysHarvestable,
+		Effective:   shovelEffective,
+		Drops:       simpleDrops(item.NewStack(g, 1)),
+		// TODO: Add Flint and drop it here.
```

Comment:
You added flints in this pull request so make sure to drop them with the right chance.